src/interfaces/IERC20Votes.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.25;
+
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
```

Comment:
really silly that oz does not provide an interface for this class , but I see no better option than what you did 

thoughts @bagelface ?

Reply:
seems fine to me